tuxemon/log.py

```diff
@@ -32,8 +32,34 @@
 import sys
 
 from tuxemon import prepare
+from tuxemon.constants.paths import *
+import requests
+import gzip
+from shutil import copyfileobj, move
+from os import remove
+from os.path import exists
+from datetime import datetime
+# For messagebox
+import tkinter as tk
+from tkinter import messagebox
 
+# For gathering system information
+import platform
 
+def archive_log():
```

Comment:
this is fine now, but just a FYI, python has a log rotation, compression, etc.  if this gets more complex, we should probably use what is in the stdlib
https://stackoverflow.com/questions/8467978/python-want-logging-with-log-rotation-and-compression